setup.py

```diff
@@ -19,7 +19,7 @@
     },
     install_requires=[
         'argparse==1.3.0',
-        'PyGithub==1.25.2',
+        'PyGithub==1.25.2-squash',
```

Comment:
from http://mike.zwobble.org/2013/05/adding-git-or-hg-or-svn-dependencies-in-setup-py/
> Update: the behaviour of pip has changed, meaning that the below probably won't work as-is (or perhaps at all). 

I don't think you need to use dependency_links. Is there anyway we can get your pygithub PR merged so we don't have to figure this out?